modin/pandas/utils.py

```diff
@@ -38,12 +62,13 @@ def from_pandas(df):
 
 
 def from_arrow(at):
-    """Converts an Arrow Table to a Modin DataFrame.
+    """
+    Convert an Arrow Table to a Modin DataFrame.
 
     Parameters
     ----------
-        at : Arrow Table
-            The Arrow Table to convert from.
+    at : Arrow Table
```

Comment:
As I understand, docstrings are edited automatically by `pydocstyle`. Why does `pydocstyle` remove space between parameter name and sign ":" in some cases, but sometimes the one doesn't remove it?
```python
example 1
at : Arrow Table # a space is not removed
example 2
val: object # a space is removed
```

Reply:
Because it's done manually. `pydocstyle` does not automatically edit docstrings.

Reply:
Now I've got that.